src/geocel/vg/VecgeomParams.cc

```diff
@@ -89,6 +89,29 @@ namespace
         } while (0)
 #endif
 
+//---------------------------------------------------------------------------//
+class VecgeomVolumeAccessor final
+    : public VolumeAccessorInterface<vecgeom::LogicalVolume const*,
+                                     vecgeom::VPlacedVolume const*>
+{
+  public:
+    //! Outgoing volume node from an instance
+    VolumeRef volume(VolumeInstanceRef parent) final
+    {
+        CELER_EXPECT(parent);
+        auto result = parent->GetLogicalVolume();
+        CELER_ENSURE(result);
+        return result;
+    }
+
+    //! Outgoing instance nodes from a volume
+    ContainerVolInstRef children(VolumeRef parent) final
+    {
+        auto&& daughters = parent->GetDaughters();
```

Comment:
```suggestion
        CELER_EXPECT(parent);
        auto&& daughters = parent->GetDaughters();
```
for symmetry with the above routine?